content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -138,6 +138,11 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```shell
    echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
    ```
+   3.1 For Ubuntu Jammy (22.04 LTS)
```

Comment:
The list formatting isn't right here.

Reply:
If the ubuntu-jammy-mirror package works OK on Debian stable and on Ubuntu 22.04 LTS, why don't we switch outright to that? xenial (Ubuntu 16.04) is no longer fully maintained.

Reply:
@sftim I'm not sure about Debian distro, I haven't used it for a long time.

About the Ubuntu if you check [here](https://ubuntu.com/about/release-cycle), the release cycle persists until 2026.

Reply:
> The list formatting isn't right here.

I will check it.

Reply:
> The list formatting isn't right here.

I've checked again and looks like okay to me, I've been following this [guide](https://www.markdownguide.org/basic-syntax/)

Reply:
> If the ubuntu-jammy-mirror package works OK on Debian stable and on Ubuntu 22.04 LTS, why don't we switch outright to that? xenial (Ubuntu 16.04) is no longer fully maintained.

About the Debian 12 (bookworm) there's the directory `keyrings`:
![image](https://user-images.githubusercontent.com/1542315/224073659-40b35859-ab7e-47fe-9194-d5877d2792f2.png)